src/control/content/components/LocationsActionBar.js

```diff
@@ -108,24 +108,56 @@ class LocationsActionBar extends React.Component {
             }).catch(() => undefined)
           );
         } else {
-          return {
-            id,
-            title:
-              typeof title === "number"
-                ? title.toString()
-                : title || "Untitled Location",
-            subtitle,
-            categories: selectedCategory,
-            address: {
-              name,
-              lat: parseFloat(address_lat),
-              lng: parseFloat(address_lng),
-            },
-            description,
-            image,
-            index: i + places.length,
-            indexForError: i
-          };
+        promises.push(
+            new Promise((resolve, reject) => {
+          let lat = address_lat;
+          let lng = address_lng;
+          let addressUrl = `https://maps.googleapis.com/maps/api/geocode/json?key=AIzaSyBOp1GltsWARlkHhF1H_cb6xtdR1pvNDAk&latlng=${lat},${lng}`
```

Comment:
you sure you want the API key to be hardly typed?

Reply:
It is the same URL as it's on fetching lat and lng from the address, i just changed the variables in the end instead of address I passed in lat and lng